app/views/shared/_filtered_card.html.erb

```diff
@@ -0,0 +1,37 @@
+<%
+  id ||= nil
+  title ||= nil
+  subtitle ||= nil
+  footer ||= nil
```

Comment:
Scott's dumb lack of Rails knowledge…what falsey value are these defaulting to such that `nil` is better? `false`, right? 🤔

Reply:
I think I grabbed these from another branch -- this partial can be cleaned / simplified a bit, but I'd like to do it elsewhere.